src/main/webapp/app/overview/course-exams/course-exam-detail/course-exam-detail.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, Input } from '@angular/core';
+import { Router } from '@angular/router';
+import { Exam } from 'app/entities/exam.model';
+import { Course } from 'app/entities/course.model';
+import * as moment from 'moment';
+
+@Component({
+    selector: 'jhi-course-exam-detail',
+    templateUrl: './course-exam-detail.component.html',
+    styleUrls: ['../course-exams.scss'],
+})
+export class CourseExamDetailComponent {
+    @Input() exam: Exam;
+    @Input() course: Course;
+
+    constructor(private router: Router) {}
+
+    /**
+     * navigate to /courses/:courseid/exam/:examId
```

Comment:
I was wondering whether this should be `/exams/` instead of `/exam/`. Whenever a student navigates to an exercise, they are routed to (e.g.) `/courses/1/exercises/1`. Changing `/exam/` to `/exams/` would help keep it consistent with the overview (`/courses/:courseId/exams`) as well.

Reply:
Yes, good point, I changed it now to `exams`